plugin-enclave-gradle/build.gradle

```diff
@@ -4,6 +4,8 @@ plugins {
 }
 
 gradlePlugin {
+    automatedPublishing = false
```

Comment:
This is to avoid a double publication of the artifact of this project. The last one (`pluginMaven`) is overriding the first one, which is created in the main `build.gradle` file